.github/workflows/check-patch.yml

```diff
@@ -0,0 +1,41 @@
+name: Check patch
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build-el8:
+
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/centos/centos:stream8
+
+    steps:
+    - name: Import ovirt-master-snapshot key
+      run: rpm --import https://download.copr.fedorainfracloud.org/results/ovirt/ovirt-master-snapshot/pubkey.gpg
+
+    - name: Install ovirt-release-master
+      run: dnf --repofrompath=ovirt-master-snapshot,https://download.copr.fedorainfracloud.org/results/ovirt/ovirt-master-snapshot/centos-stream-8-x86_64/ install -y ovirt-release-master
+
```

Comment:
and update above accordingly, the rpm to be installed will be ovirt-release44-snapshot